.travis.yml

```diff
@@ -6,15 +6,24 @@ services:
   - docker
 
 install:
-  - pip install "ome-ansible-molecule<0.5"
-
+  - >-
+      if [ "$PYTHON" = "2" ]; then
+          pip install "ome-ansible-molecule<0.5";
+      fi
+  # Ansible 2.6 has problems when the remote python version is 3
```

Comment:
Now that the work for unifying up and testing all the [OME Ansible roles](https://github.com/ome/ansible-roles) using `ome-ansible-molecule` has been completed, is it worth coming back to https://github.com/ome/ome-ansible-molecule/pull/20 and move towards a new release of the meta-package?